dali/operators/reader/video_reader_op.h

```diff
@@ -132,16 +124,11 @@ class VideoReader : public DataReader<GPUBackend, SequenceWrapper> {
     }
   }
 
-  virtual void ProcessSingleVideo(
-    int data_idx,
+  virtual void ProcessVideo(
     TensorList<GPUBackend> &video_output,
-    SequenceWrapper &prefetched_video,
+    TensorList<GPUBackend> &video_batch,
     DeviceWorkspace &ws) {
-    video_output.type().Copy<GPUBackend, GPUBackend>(
-      video_output.raw_mutable_tensor(data_idx),
-      prefetched_video.sequence.raw_data(),
-      prefetched_video.sequence.size(),
-      ws.stream());
+    video_output.Copy(video_batch, ws.stream());
```

Comment:
This indentation (same for args and code) is quite confusing. Can you autoformat?

Reply:
Done